keda/README.md

```diff
@@ -109,6 +109,11 @@ their default values.
 
 | Parameter | Type | Default | Description |
 |-----------|------|---------|-------------|
+| `autoscaling.verticalPodAutoscaler.keda.cpu.maxAllowed` | int | `2` |  |
```

Comment:
This is automatically generated though: https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#documentation

Reply:
Do you mean I can get rid of this change in the PR as it will be automatically be run by the ci jobs ?